pom.xml

```diff
@@ -13,8 +13,8 @@
     <properties>
         <maven.compiler.target>1.8</maven.compiler.target>
         <maven.compiler.source>1.8</maven.compiler.source>
-        <junit.version>5.7.1</junit.version>
-        <hibernate.version>5.4.31.Final</hibernate.version>
+        <version.junit.jupiter>5.7.1</version.junit.jupiter>
```

Comment:
Convention for property names that keeps version is rather: `<dependency.name.version>`